### PR TITLE
bugfix: add DSS accredited non-DOD systems banner

### PR DESCRIPTION
**Pull request by dirtyharrycallahan**
* Adds the banner for DSS accredited Non-DoD systems, systems not
  connected to SIPRNet

* Reference
  - ODAA Process Manual, Version 3.2, November 13, 2013
  - http://www.dss.mil/documents/odaa/ODAA%20Process%20Manual%20Version%203.2.pdf

### CODE REVIEW

**Comment by redhatrises** [comment]
Thanks for the patch. "non_dod_default" should be changed to "dss_odaa_default". Otherwise, once that is changed, this can be merged.

**Comment by redhatrises** [comment]
Also, add the banner to RHEL/7 content as well.

File: RHEL/6/input/system/accounts/banners.xml [hunk 1]
```diff
@@ -24,6 +24,7 @@ be expressed by the '\n' character and special characters like parentheses and q
...
...]+of[\s\n]+this[\s\n]+or[\s\n]+any[\s\n]+other[\s\n]+DoD[\s\n]+interest[\s\n]+computer[\s\n]+system[\s\n]+reveals[\s\n]+possible[\s\n]+evidence[\s\n]+of[\s\n]+violation[\s\n]+of[\s\n]+criminal[\s\n]+statutes,[\s\n]+this[\s\n]+evidence[\s\n]+and[\s\n]+any[\s\n]+other[\s\n]+related[\s\n]+information,[\s\n]+including[\s\n]+identification[\s\n]+information[\s\n]+about[\s\n]+the[\s\n]+user,[\s\n]+may[\s\n]+be[\s\n]+provided[\s\n]+to[\s\n]+law[\s\n]+enforcement[\s\n]+officials.[\s\n]+If[\s\n]+monitoring[\s\n]+of[\s\n]+this[\s\n]+or[\s\n]+any[\s\n]+other[\s\n]+DoD[\s\n]+interest[\s\n]+computer[\s\n]+systems[\s\n]+reveals[\s\n]+violations[\s\n]+of[\s\n]+security[\s\n]+regulations[\s\n]+or[\s\n]+unauthorized[\s\n]+use,[\s\n]+employees[\s\n]+who[\s\n]+violate[\s\n]+security[\s\n]+regulations[\s\n]+or[\s\n]+make[\s\n]+unauthorized[\s\n]+use[\s\n]+of[\s\n]+DoD[\s\n]+interest[\s\n]+computer[\s\n]+systems[\s\n]+are[\s\n]+subject[\s\n]+to[\s\n]+appropriate[\s\n]+disciplinary[\s\n]+action.[\s\n]+Use[\s\n]+of[\s\n]+this[\s\n]+or[\s\n]+any[\s\n]+other[\s\n]+DoD[\s\n]+interest[\s\n]+computer[\s\n]+system[\s\n]+constitutes[\s\n]+consent[\s\n]+to[\s\n]+monitoring[\s\n]+at[\s\n]+all[\s\n]+times.</value>
```
Comment (shawndwells) [inline RHEL/6/input/system/accounts/banners.xml]: "non_dod_default" implies commercial sector to many users. Perhaps "dod_interest" would be acceptable?

**Comment by shawndwells** [comment]
Should this be dropped into RHEL7 as well?

File: RHEL/6/input/system/accounts/banners.xml (hunk 1 above)
Comment (redhatrises) [inline RHEL/6/input/system/accounts/banners.xml]: @shawndwells 'dss_odaa_default' may make more sense.

Comment (shawndwells) [inline RHEL/6/input/system/accounts/banners.xml]: On 8/14/15 2:44 PM, redhatrises wrote:
> @shawndwells <https://github.com/shawndwells> 'dss_odaa_default' may 
> make more sense.

works for me :) something besides "non_dod" is the goal.

**Comment by dirtyharrycallahan** [comment]
As you probably know the ODAA reference includes a SIPRNet (DoD) banner and a Non-DoD banner for systems not connected to SIPRNet. In this context dod_default lacks the reference to SIPRNet but so does dss_odaa_default. "Non-DoD" was taken from the ODAA reference. "dod_non_siprnet_default?" My frame of reference does not extend beyond my experience with DSS so I can live with whatever you suggest. We do not have RHEL 7 yet but it is coming so I can drop the change in there as well. There did not seem to be an easy way to not repeat content. Perhaps I need to dig a little deeper before violating DRY.

thanks for looking at the pull request.

**Comment by redhatrises** [comment]
@dirtyharrycallahan please do add the banner to `RHEL/7` as well as name it `dss_odaa_default` as SIPRNET will use the `dod_default` and STIGS anyway.

**Comment by dirtyharrycallahan** [comment]
Sorry for the delay. There was an internal discussion about IP and whether I could do this and get paid by my company. IP is not an issue but I will have to do this on my own time for now. I should be able to get to it today or tomorrow.

**Comment by shawndwells** [comment]
@dirtyharrycallahan thank you for submitting this! Your patch has been lingering for awhile. Merging manually with the following changes:

`````
$ git commit -a -m "renaming RHEL6 non_dod_default banner to dss_odaa_default"
[dirtyharrycallahan-odaa_banner 4716ce2] renaming RHEL6 non_dod_default banner to dss_odaa_default
 1 file changed, 1 insertion(+), 1 deletion(-)
`````
and
`````
$ git commit -a -m "Adding dss_odaa_default banner to RHEL7"
[dirtyharrycallahan-odaa_banner 3c79848] Adding dss_odaa_default banner to RHEL7
 1 file changed, 1 insertion(+)
`````

**Comment by shawndwells** [comment]
(to be clear - this means the dss_odaa_default banner will be selectable when using an SCAP editor like SCAP Workbench. this does not add it to any existing profiles as selected value)

**Pull request merged**